llvm/lib/CodeGen/CodeGenPrepare.cpp

```diff
@@ -8359,6 +8391,30 @@ bool CodeGenPrepare::fixupDbgValue(Instruction *I) {
   return AnyChange;
 }
 
+bool CodeGenPrepare::fixupDPValue(DPValue &DPV) {
```

Comment:
The debug intrinsic path asserts that it's a `DbgValueInst`. `fixupDbgValue` is only called on dbg_value (and dbg_assign)-type intrinsics, whereas `fixupDPVAlue` is called without first filtering by DPValue kind. This is fine - no action required.